sunpy/map/mapbase.py

```diff
@@ -246,12 +249,12 @@ def __repr__(self):
                                tmf=TIME_FORMAT) + self.data.__repr__()
 
     @classmethod
-    def _new_instance(cls, data, meta, plot_settings=None, **kwargs):
+    def _new_instance(cls, data, meta, **kwargs):
         """
         Instantiate a new instance of this class using given data.
-        This is a shortcut for ``type(self)(data, meta, plot_settings)``.
+        This is a shortcut for ``type(self)(data, meta)``.
         """
-        return cls(data, meta, plot_settings=plot_settings, **kwargs)
+        return cls(data, meta, **kwargs)
```

Comment:
So given our removal of `cmap` and `norm` from the constructor, in order to preserve the cmap and norm across new instance creation, what I think we want here is

```suggestion
        new_instance =  cls(data, meta, **kwargs)
        new_instance.cmap = self.cmap
        new_instance.norm = self.norm
        return new_instance
```